js/apps/system/_admin/aardvark/APP/react/src/views/databases/addDatabase/AddDatabaseModal.tsx

```diff
@@ -0,0 +1,121 @@
+import { Button, Flex, Heading, Stack } from "@chakra-ui/react";
+import { Form, Formik } from "formik";
+import React from "react";
+import { mutate } from "swr";
+import * as Yup from "yup";
+import {
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalHeader
+} from "../../../components/modal";
+import { getCurrentDB } from "../../../utils/arangoClient";
+import { AddDatabaseForm } from "./AddDatabaseForm";
+import { DatabaseDescription } from "../Database.types";
+import { useFetchInitialValues } from "../useFetchInitialValues";
+
+export const AddDatabaseModal = ({
+  isOpen,
+  onClose
+}: {
+  isOpen: boolean;
+  onClose: () => void;
+}) => {
+  const initialFocusRef = React.useRef<HTMLInputElement>(null);
+  const { defaults: INITIAL_VALUES } = useFetchInitialValues();
```

Comment:
can can name this initialValues, as I've generally been using all-caps for static values, can get confusing

```suggestion
  const { defaults: initialValues } = useFetchInitialValues();
```